net/remote/remote-client.sh

```diff
@@ -55,7 +55,6 @@ clientCommand="\
     --network $entrypointIp:8001 \
     --drone $entrypointIp:9900 \
     --duration 7500 \
-    --sustained \
```

Comment:
Maybe we want to do the opposite, i.e, if --sustained don't do barriers. Since with sustained we're probably going for perf. Maybe ideally we want a "--bench" ?